models/organization.py

```diff
@@ -48,6 +48,7 @@ class Organization(SerializableModel):
     city = models.CharField(max_length=255, blank=True, null=True)
     country = models.ForeignKey('reference.Country', blank=True, null=True)
     cohort = models.ForeignKey('internship.Cohort', null=False, on_delete=models.CASCADE)
+    address = [location, postal_code, city, country]
```

Comment:
I didn't get this change. Is it related to the problem ?

Reply:
An error was preventing me to enter the information page for the student as the template was looking for organization.address.location, organization.address.postal_code, etc. and address was undefined. 

Therefore, the change was either to make 4x in the template or once in the model.

Reply:
Ok, but I can't find where you are using the variable `address`.

Reply:
https://github.com/uclouvain/osis-internship/blob/17242086d0a200a1ca3c6403d0709d1d3f4aabf5/templates/student.html#L144-L149

But it has been changed lately so we should indeed not consider this line.

`address = [location, postal_code, city, country]`